### ISSUE TITLE
Removing 3x3x3 With Feet as an official event

### ISSUE THREAD

**Issue by lgarron** [thewca/wca-regulations#652]
Removing 3x3x3 With Feet from the list of official events is regularly requested. 

In September 2017, a Delegate poll with the question "Should 3x3x3 With Feet be an official WCA event?" resulted in the following votes:
- Yes: 56
- No: 58

In November 2017, the Board agreed with a proposed plan:

> The format of Feet should be changed from Mean of 3 to Average of 5. One year after the change has come into effect, we will analyze the popularity again. Feet will only be kept as an event if more than 30% of the competitions in the last 12 months held Feet and more than 4.5% of the competitors competing in the last 12 months competed in Feet.

The Ao5 change was #479. The previous discussion for removing 3x3x3 With Feet was #175.

2018 is not over yet, but the numbers are not on track for the proposed plan. Removing 3x3x3 with Feet in early 2019 is very short notice, so the overall WRC preference at the moment is to remove 3x3x3 With Feet from the list of official events in 2020.

(Note: This post was edited to include details about the Delegate poll from last year, since many people were assuming there was no vote.)

**Comment by pedrosino** [comment]
Is this independent on the 2019 numbers?
I could see people arguing that it should stay if the numbers go up next year.

**Comment by UnsolvedCypher** [comment]
I'm not very happy with how this is being handled- this is an issue that would affect the WCA community at large, and it seems that this decision was made by the WRC and the Board without community input. I would strongly support an effort to gather feedback from the community on this change. If there is a desire to remove this event, I think the conditions for its removal (such as the 4.5% of competitors threshold) should be debated and disclosed in advance in a manner that's transparent to the community.

**Comment by samsschultz** [comment]
I second Matthew's concern. I believe the conditions for keeping 3x3x3 With Feet as an official event (30% of 2018 competitions, 4.5% of 2018 competitors having competed at least once in the event) should have been disseminated to the community in advance. As an organization, the WCA should be transparent about decisions and situations like this, as their outcomes will have a direct and noticeable effect on community members. 

Keeping those conditions behind closed doors until recently is an example of a lack of transparency. I don't see a major problem with disseminating those conditions to allow the community to understand the process and to provide feedback relating to it. 

 If the WCA does not address the larger issue of lacking transparency in decision informing processes that affect community members, I fear it could lead to a long term impediment of growth and interest in the organization, especially as more organizational fragmentation is starting to occur in speedcubing at large.

**Comment by pedrosino** [comment]
I believe the reasoning in not disclosing those conditions was to avoid inflated numbers. People who really care about the event would have pushed hard on organizers to hold it, creating a "false" demand and non-natural growth/numbers.

**Comment by tobby125** [comment]
The 30% requirement doesn't make much sense to me. This year FMC, 4BLD, 5BLD, and MBLD were all held at less than 30% of competitions. Although there may not have been many new competitors, there is still an apparent increase in popularity, as many people are now taking the event much more seriously. 66% of the top 100 feet average rankings were just set this year with an average of 5. At the end of 2017, 4 people had sub 30 averages, now 11 people do. I can only see the popularity of feet growing even more, and it's very unfortunate that so many people that have dedicated so much time to the event, especially those that gained interest this year, will now see all their hard work go to waste.

**Comment by samsschultz** [comment]
I understand that reasoning for those specific metrics, though Delegates already have the responsibility of ensuring all events are proportionally included in their region's competitions relative to regional popularity and demand, so a Delegate could have simply refuted such pressure to hold the event at a particular competition if it wouldn't have been appropriate. 

I also believe the entire process should have been made more public, as there has yet (to my knowledge) to be publicly disseminated information explaining, rationalizing, or otherwise justifying the decision to remove (or explore the removal of) the 3x3x3 With Feet. The WCA should be trying to serve the community as well as possible, and I therefore believe obtaining an accurate community perspective to understand the ramifications of such a big decision is essential.

**Comment by CarterKoala** [comment]
Yes, I would love to see more about how this decision came about.

This seems really random and unexpected at all. I haven't even seen any [good] arguments for why it should be removed. The main argument I've seen is that it's gross, which is just not true compared to your hands.

One thing I also don't get is why the WRC is voting on this. Having 6 people vote on removing an event, where most are biased against it, is just stupid IMO. 

WRC members:
-Competes consistently, has had 9 NRs in the event
-4:30 single, hasn't competed since 2013
-7:23 single, hasn't competed since 2012
-3:16 mean, hasn't competed since 2015
-6:46 mean, competed 3 times in 2016
-Never competed
-Never competed

**Comment by jamesquinn1** [comment]
To start: I am strongly against the removal of feet, and this is a hill I will die on.

Here are some of my reasons:

Unpublished rates:
As far as I know (and I very well might be wrong) the 30% of competitions must host feet was never posted, and I believe that more competitions would have hosted the event in order for it to be kept.
Unfair rates:
I believe that the 30% was unfair, as at least 4 other events have below 30% as well. Also, feet hadn't been over 25% since 2013, so expecting it to hit 30% wasn't going to happen, and in my opinion the board was intending an hoping to remove it. It did increase however.
Unfair voting:
I believe that for an event removal, there should be more than 6 people voting. Also, only one of those people has competed in feet since 2016, and they are also the only one who's times show more than a day of practice.
Removal of events in the past
Only 2 events have been removed that were held for a decent length of time, the magics, and they had to be removed, not just some people wanted them to be removed. From my understanding they were removed because they weren't twisty puzzles, were impossible to judge, likely weren't being held under the same circumstances all around the world, and created many incidents. None of these apply to feet.
Changing of format unfair
I believe that changing the format one year ago was unfair to competitors if the wca was intending to remove the event, as changing the format to more solves seemed to give of the impression of feet being a more serious event, and that it was here to be kept. This caused people to put a lot more practice into the event, and that time can seem wasted now.
Why I think the event is held so rarely:

Equipment
Delegates often don't want their equipment used for feet, and will prohibit organizers who want to hold the event from having it. Yes, organizers can get other equipment, but this adds a step in the process and I believe many organizers would just scrap the idea.
I also believe that this leads to less people getting to compete, and subsequently getting into the event, and wanting to compete more, and hosting it.
Moving forward:
I believe that the event should be kept. I am personally fine with changing the event back to mo3 so it takes less time and can be held more. I just want to be able to keep doing the event.

In closing,
I think that people need to be more aware of feet. I know lots of non-cubers who are more impressed by feet than by standard solves. I think that Daniel Rose-Levine and others are finally bringing exposure to the event and as such people are considering doing it. Removing it now while it is really beginning to show where it could go I don't think it's the correct idea. The world record has improved a lot lately and from what DRL told me he averages, it will improve a lot more. I also think it is an event where people who aren't usually considered super elite can win as it is a very different kind of skill, and that gives more people chances to podium and set records, encouraging them to cube more.

**Comment by casiku** [comment]
From what I can read in these github issues, I don’t think the board really understands what it takes to be a feet solver. 3x3 WF is an event that requires few moves and therefore logic, lookahead and many algorithms. You can’t just spam 60 moves to become fast, like you can in two- and one-handed, since TPS maxes out at ~2 TPS. I learned ~200 algorithms for feet, and because of the nature of the event, these algorithms are useless in other events like two- and one-handed. These algorithms are near-optimal and involve mostly R/F/B/U in an order that would be very awkward to execute with hands.

**Comment by Kymlikespotatoes** [comment]
I would have never imagined possibly  removed so soon. It might have not hit the 30% mark (which I don't agree with, other events were lower) , but it has improved significantly in the past year. More than half of the results in the top 100 were from this year and will continue to improve. Before 2018 sub 30 was top 5 in the world and now its not even top 10. The world record average was also broken more times this year alone then it was in 2016-2017 ( and those didn’t even drop as significantly as they did this year). Another issue with this proposal is that 6 people ( 5 of which are very inexperienced with feet) are deciding on this for the entire community, this seems quite biased and the competitors should have some say. I also can’t really think of reasons on why it should be removed, all I’ve heard are “the event is gross” or “ it’s a pain to organize” which aren’t very good reasons and can easily be fixed. In conclusion I would hate to see so many people’s time on this event go down the drain. You could say the same about magic, but magic actually had convincing arguments.

**Comment by tnorr** [comment]
None of this makes sense to me. The numbers clearly tell that feet has gained a much wider popularity this year than before. Setting an arbitrary limit to remove it as an event, behind the community's back, requiring the popularity to rise much more than expected, during one year when a new format has just been introduced is just plain misuse of board's power. It's not possible to make the process more biased when it's clear that the people voting are the people who couldn't personally care less about the event. The least that should be done is to follow the change in popularity for at least another year AND definitely implement a way to use the opinion of a wider community.

As to the organizational issues, it's up to the organizers to worry about those. There's not much to worry about but if it's a huge problem, no one is required to hold the event. Why should it be a reason to ban everyone from holding it?

Thank you, makes me lose a lot of respect for the WCA.

**Comment by Sixstringcal** [comment]
I personally think this should have been more open to the community and more transparent.  People should have known that if they organized a comp with feet it could have helped save it.  I get there's a bias, but it allows organizers that are interested in feet, or at least don't want to see feet go for one reason or another can do something.  This news was very sudden and unexpected.  As an event, I think it's a fun and refreshing event.  It's not my main event and it's not an event I practice much, but I compete in it whenever I can and I think it's cool watching people solve it way faster than I can with their feet.  It's a very difficult thing solving a cube with your feet, and people have worked so hard to improve.  As stated before, many of their algs don't make sense to use anywhere else.  These people have dedicated so much time to this only for it to be irrelevant.

**Comment by zzwerling** [comment]
There needs to be a better voting system. As carter said, having 7 people decide, most of which who clearly have not practiced feet (according to their results), is unfair. I think the WCA should implement some sort of system allowing the community to vote. This would require a lot of time and effort to create a system that works and isn't biased, but I believe something could be achieved, assuming that some restrictions are placed in order to prevent nonsensical voting.

**Comment by kingmathyall** [comment]
@jamesquinn1 

I completely agree that feet should not be removed. You bring up very good
points that I 100% agree with and I believe this should at the very least
be decided by a community vote.

**Comment by jamesquinn1** [comment]
Also, for the announcement of the removal of master magic, the WCA supplied reasons for the deletion.

(From the wca forum [post](https://www.worldcubeassociation.org/archive/forum_topics/1013))
As a result of discussions between the WCA Board and the WCA delegates, the board has decided to remove Magic and Master Magic from the list of WCA sanctioned events at the end of the year 2012. Starting on January 1, 2013, the WCA will no longer accept or sanction events and times for Magic and Master Magic. Whereas we realize that many members of our community are extremely dedicated and passionate regarding these events, we have decided to remove them for the following reasons: 1. Lack of puzzle quality - Magic and Master Magic are rote processes that require no thinking. The events are simply a measure of a competitor's ability to perform a unchanging set of physical moves. Although other events, such as 3x3x3 speed solve does have elements that are similar to repetitive physical tasks, the lack of an analysis element in Magic and Master Magic make them less creative activities. 2. Difficulty in regulation - The nature of Master and Master Magic is such that the start and the finish are critical to the competitor's time. As such, whether intentional or unintentional, it is very difficult for even experienced judges to determine whether the solve's start or finish has been performed in accordance to regulations. The WCA has reason to believe that these events have not been held under consistent circumstances around the world. The WCA emphasizes that these inconsistencies are as a result of the nature of the event more than the experience of the officials. Magic and Master Magic events will continue to be held at WCA events throughout the end of 2012, at which time the WCA will archive the results of these events. If you have any questions, concerns, or comments, please contact your local WCA delegate. Regards, Tyson Mao

This also went to a delegate vote.

**Comment by cubewhiz** [comment]
When the WRC provided the benchmarks to the WCA Board last year, we evaluated whether it would be better to provide or withhold them. It was determined that withholding the benchmarks will provide us with more natural numbers and avoid people hosting competitions with 3x3x3 with feet just for the sake of trying to save the event (and likewise to prevent those who want to see the event removed from not allowing it to be held). Comments like, "People should have known that if they organized a comp with feet it could have helped save it." and "As far as I know (and I very well might be wrong) the 30% of competitions must host feet was never posted, and I believe that more competitions would have hosted the event in order for it to be kept." point out precisely the reason we did not publish the benchmarks. While there was no community vote (nor do we have the means in place to have one yet), there was a vote among Delegates last year.

**Comment by jamesquinn1** [comment]
@cubewhiz 
Bob,
Thanks for clearing that up.
Was there a delegate vote recently, to determine whether to remove the event, or was it just the WRC?

**Comment by cubewhiz** [comment]
@jamesquinn1 There was a Delegate vote in 2017 regarding this. There was not one in 2018.

**Comment by ccbaird** [comment]
@cubewhiz Is there any plan to have a delegate vote on the issue? It may be naive of me, but I'm inclined to think that the apparent improvement in the level of competition, if not in raw numbers, may warrant at the very least an extension of the probationary period.

For what it's worth, I for one really enjoy WF and have competed in the past, but during 2018 I had only one competition that was near enough and without a schedule conflict that held WF. I got about 5 miles down the road on the way there and the brakes on my car went out. I decided against driving another 80 miles without brakes, and skipped the competition. I look forward to competing in WF at least one more time at the competition I am organizing in February, and sincerely hope it's not the last time I have the opportunity to do so. I will comment later with the arguments that I personally find compelling for keeping WF as an official event, but I need to sleep now. Thanks!

**Comment by Joliv3r** [comment]
Removing feet as an event will be a huge loss. Feet is a unique event (since it's the only event where you can't use your hands) and it keeps variety in the WCA. Feet is also different from the other 3x3 events because you have to be both efficient and fast. Since it's really hard to turn very fast at feet, a lot of the techniques and algs people have learned especially for feet will go to waste. This because they're not any good for other 3x3 events (maybe except FMC). 

I can't see any good reasons to remove feet either. As mentioned in other comments, 4BLD, 5BLD, MBLD, and FMC did not meet the requirements set for feet. It's unrealistic to think feet would meet these requirements both because it never has and other events also don't meet them. As I understand in the removal of magic and master magic, is that it was necesarry because it required no thinking, and that it essentially was just doing the same thing over and over again. Feet on the other hand is not anything like this. It requires more thinking to do feet than normal 3x3 because it's so important to be efficient. Magic and master magic was also difficult to judge and was held under different circumstances at different competitions. This is also not the case for feet, since it's judged in the same way as normal 3x3.

In conclusion, there is no reason to remove feet as an event. It's different from other events, which make you more skilled if you're good in every event. It's also different from normal 3x3 solving, because you have to be more efficient and can't just spam tps. None of the reasons magic and master magic was removed applies to feet. It's definitely a solve and not just the same procedure over and over, and feet is not hard to judge. That it's "gross" is not a reason to remove feet.

**Comment by pajwo** [comment]
>           There needs to be a better voting system. As carter said, having 7 people decide, most of which who clearly have not practiced feet (according to their results), is unfair. I think the WCA should implement some sort of system allowing the community to vote. This would require a lot of time and effort to create a system that works and isn't biased, but I believe something could be achieved, assuming that some restrictions are placed in order to prevent nonsensical voting.

How is that even relevant? They are mature people, don't expect from them practicing 3x3 with feet. The case of removing feet as an official event was brought up many times on speedsolving forum and every time majority of people was against keeping it. There is no need for letting community vote as the WCA Board clearly knows what they are doing and they are doing it because we wanted it. It's a result of years of complaining about this event and I am really glad they finally started to act.

The only issue I can see is that if feet will be removed, Clock should go as well for a number of reasons. That would be 4th event getting removed (not counting mbld old style and that weird diamond cube). Really, I don't mind it, but it would be nice to see some events being added. Unfortunately it's hard to find an unique puzzle which isn't similar to current events. People often mention Team BLD and Kilominx though.

**Comment by tnorr** [comment]
It seems to me there have been a lot of issues with feet and clock being too different. Why is the goal finding a new event that is unique and not similar to current events then?

**Comment by lgarron** [comment]
I'd like to thank everyone for taking the time to weigh in here –  I appreciate the constructive tone.

As mentioned in the first post, the WRC does not plan to recommend removing 3x3x3 with Feet in Jan. 2019, which means we have more time to discuss the process.

From the response to this issue, it sounds like a lot of people feel like there should be more voices contributing to this particular decision. While it seems that the community is (and will remain) fairly divided on this, I would definitely love to see a community-wide survey to understand competitor's opinions and preferences. The WCT is now interested in doing such a survey, so I hope we can make that happen.

Reasons that people do/don't like the event are fairly understood (#175), so I think it would be most useful to discuss how to get a representative view of the entire community. To this end, it would be useful to work towards a set of questions that are appropriate to ask about what people think of 3x3x3 With Feet and what should be done with its official status.

**Comment by jamesquinn1** [comment]
"How is that even relevant? They are mature people, don't expect from them practicing 3x3 with feet."
When the voting members aren't qualified to make the vote, it's bias.

"The case of removing feet as an official event was brought up many times on speedsolving forum and every time majority of people was against keeping it."
I found 3 polled thread on ss about removing feet
https://www.speedsolving.com/forum/threads/should-3x3x3-with-feet-remain-an-official-wca-event.48355/
44% keep, 36% remove, 19% no opinion
https://www.speedsolving.com/forum/threads/3x3-feet-as-an-event.63754/
71% keep, 18% remove
https://www.speedsolving.com/forum/threads/remove-3x3x3-with-feet-as-an-official-event.43591/
48% keep, 19% remove, 32% dont care.

I think the majority of people are for keeping the event based on ss numbers.

"There is no need for letting community vote as the WCA Board clearly knows what they are doing and they are doing it because we wanted it."
The community needs a vote. Also, WCA dues make this taxation without representation :(

**Comment by ccbaird** [comment]
@jamesquinn1 it is worth noting the dates of those polls:
48% remove, 32.3% keep, 19.7% don’t care August 13, 2013
36.5% remove, 44.4% keep, 19.2% no opinion July 6, 2014
28.4% remove, 71.6% keep February 2, 2017

The polls have different options, and aren’t perfectly comparable, but that looks like a trend toward sentiment favoring keeping feet as an official event to me.

**Comment by tobby125** [comment]
I don't think people's sentiment towards the event in 2013 should really affect any decision now. I know a lot of people that used to think feet was stupid, but started practicing this year, and now they really like it. I only think this trajectory will continue.

In terms of whether or not the WRC members are "mature" people, I'm sure they are. They are making the decision that they feel is best and that is fine. However, the WCA is a community organization, and I believe that if most of the community wants something to happen, it should. 6 people couldn't possibly represent the 100,000+ people in the WCA.

**Comment by Tehzeebkohli** [comment]
I really hope the WCA takes into consideration the passion and hard work put into by the feet solvers. We have witnessed such fabulous results this year. 
As a community we need to be inclusive, kind, considerate and sensitive to other people around us.
 A decision like this would leave so many feet solvers heartbroken and demotivated. 
Keep feet and clock. There are plenty of clock fanatics around as well as die hard fans of feet solving. 
Some people may not like either but that doesn’t take away the love and passion for these events which so many members of the community have.

**Comment by lgarron** [comment]
Since many people are assuming this was "just a vote of 6 people", I've added details to the first post in this issue about last year's Delegate poll.

**Comment by Joliv3r** [comment]
I don't think we should remove an event because of a vote which was almost a tie (56 to 58). In the removal of an event it should be clear problems with the event and a big majority should be against it. Removing an event because of a 56 to 58 vote seems rediculous to me. This is 49.13% to 50.87%.

**Comment by tnorr** [comment]
^ completely agree, this is so ridiculous

**Comment by CarterKoala** [comment]
Yeah, if there’s a vote on such a big change as this, then there should have to be overwhelming support to change it for it to be changed, 75% at minimum. 50.87% is just crazy...

**Comment by lgarron** [comment]
> I don't think we should remove an event because of a vote which was almost a tie (56 to 58).

Indeed, this is what led to the change in format and the evaluation proposal.

**Comment by ccbaird** [comment]
Am I understanding correctly that the initial “should it be an event” question was voted on by delegates and effectively tied, but the format change and conditions for continuing were only voted on by the board?

**Comment by jamesquinn1** [comment]
Also, I think a delegate vote now is mandatory at a minimum. That proposal was a year ago, and a lot as changed, including delegates minds. There is also many new delegates. For the event to be removed, I feel another delegate vote is needed, with a percentage way above 50.8%.
A community vote is also needed.

**Comment by CarterKoala** [comment]
> Since many people are assuming this was "just a vote of 6 people", I've added details to the first post in this issue about last year's Delegate poll.

This doesn’t help the fact that 6 people made the final decision on something as big as removing an event.

**Comment by lgarron** [comment]
> This doesn’t help the fact that 6 people made the final decision on something as big as removing an event.

Please note that there has been no final decision yet. That would be when the Board decides to accept concrete Regulations changes proposed by the WRC (which will not happen this year).

**Comment by ccbaird** [comment]
@jamesquinn1 is there precedent for a community vote? I definitely agree there should be a delegate vote, I’m just a little concerned about the logistics of having potentially tens of thousands of people vote on an issue that objectively a majority have no opinion one way or the other on.

**Comment by CarterKoala** [comment]
@ccbaird I know a lot of people in Cyoubx’s Friends were talking about it, but there wasn’t really a good way that anybody has thought of to do it.

**Comment by lgarron** [comment]
Also, I appreciate that people are continuing to send constructive comments here and via email!

I don't want to promise that anything particular will/won't happen based on the feedback we get, but this public GitHub issue is not the final step in the process and we are listening!

**Comment by lgarron** [comment]
> Am I understanding correctly that the initial “should it be an event” question was voted on by delegates and effectively tied, but the format change and conditions for continuing were only voted on by the board?

Correct!

**Comment by jamesquinn1** [comment]
@ccbaird

There is currently nothing that I know of, but the WCT I've heard is working on it. I drafted some very rough guidelines on who can vote, generally based on activeness at competitions with a friend of mine. I can post that once we finish if anyone's curious.

**Comment by jamesquinn1** [comment]
I'm not sure if the wca will consider this, but I made a petition yesterday mostly out of curiosity and it is currently at 420 signatures. I don't think the wca has ever considered one before, but this one has gotten more signatures than most other cubing ones, and I think it should be considered.

Most of you have probably already seen it but here's the link:
https://www.change.org/p/world-cube-association-keep-the-3x3x3-with-feet-event-official

**Comment by Tehzeebkohli** [comment]
The WCA made a fabulous decision by changing the format in the beginning of 2018. The crazy records being set speak for themselves. Maybe the number of participants have increased only slightly but the fact remains that they have increased. Take into consideration that this new format has given rise to quite a number of new participants who are passionate and enthusiastic about feet solving. 
Every great change takes time.... after motivating the participants in 2018 with the new format and now giving a deadline of removing this event in 2020 totally defeats the purpose, in my humble opinion. 
Firstly I feel no event should be removed be it clock or feet because it is loved by a large group even though they may not be a majority. 
Secondly removing feet in 2020 will demotivate so many cubers who will no longer want to participate in their favourite event and fall more in love with it only for it to be taken away from them in 2020 leaving them heartbroken. Think about it!!!
This is just my humble opinion... why not give feet the encouragement it deserves, which was so rightly done by the WCA in 2018 and watch for 4-5 years to see where it goes. Then maybe a discussion and a vote can be taken. 
Every great change takes time... it starts off a trickle only to grow from strength to strength. Feet has given us some thrilling moments after the change in format which only shows that this change was so motivating for feet solvers. 
The deadline of 2020 will ruin whatever good that we’ve seen in 2018 after the new format. 
As a community it’s important to listen to the minority as well. That’s what makes us a community, that’s what makes us a global family.

**Comment by cubewhiz** [comment]
> > Am I understanding correctly that the initial “should it be an event” question was voted on by delegates and effectively tied, but the format change and conditions for continuing were only voted on by the board?
> 
> Correct!

Actually, only partially correct. The format change was voted upon by the Delegates, too, but the decisions were proposed by the WRC and agreed upon by the WCA Board.

> Yeah, if there’s a vote on such a big change as this, then there should have to be overwhelming support to change it for it to be changed, 75% at minimum.

Do you support the change that was made to change the format of feet from mean of 3 to average of 5? In that vote, there was more support for mo3 than ao5, but we made the change anyway. Sometimes, we make decisions even without an overwhelming majority.

**Comment by jamesquinn1** [comment]
I don't really care between mo3 and ao5, but if I had to pick I'd go for ao5. A change like that I think doesn't need an overwhelming majority, but removing an event entirely does.

**Comment by CarterKoala** [comment]
Yeah, as James said, that’s a bit different. I don’t necessarily agree with that if there was more support for ao5 over mo3, although I do remember quite a lot of people wanting it to be ao5.

**Comment by Tehzeebkohli** [comment]
Making it Ao5 was a fabulous decision and has positively impacted the feet solvers. The thrilling records speak for themselves. I really hope the WCA would continue encouraging feet like they did with the format change and see how amazing feet solving eventually becomes. I’m absolutely positive about this!!!!
No participant wants to feel like his favourite event is no longer going to be there in a year. It’s just so sad. 
Everything is not about numbers, (even though the numbers for feet solving after the new format clearly support the event) Emotions, passion and hard work should also be considered as well as so many dreams which will be shattered if feet is taken out in 2020 
I

**Comment by pd159** [comment]
I'd be so happy if feet gets removed. It's horrible to host at comps due to sanitary issues and is overall an abomination to not only inside the cubing community but also to the perception of the hobby by non-cubers.

It's funny how some people are complaining about not having known about the 30% threshold for last year; they had a reason for not publicly disclosing that. If they announced it then the number of feet comps would have risen artificially, which is obviously bad

**Comment by Aimankoli** [comment]
Feet is such a unique event and so much fun to watch. Many people call it weird, disgusting, gross, etc, but it's a really cool skill to have. And feet requires you to be more efficient than any other event. 
Although more people may not have competed in the event, it has definitely become more popular as we can see by the results people have been getting. We have 3 sub 20 singles now, and in 2017 we had none. The amount of sub 30 averages have tripled. It would be bad to see this event go when the results are becoming so much better

**Comment by lgarron** [comment]
I'd like to thank everyone for continuing to send constructive comments to the WRC!

To reiterate the first post, the WRC is not pursuing this change for the Jan. 2019 Regulations.
We (and the Board) are discussing the best way to continue to evaluate 3x3x3 With Feet during 2019 – in particular, how to get an appropriate view of the community's preferences – but I don't have more to share at this moment.

**Comment by J-Rohault-dF** [comment]
Again, the most important decisions (adding or removing an event) should need a supermajority (either 60%, 66.6% or 75%, I'll let you decide).
It should maybe also need an overall majority in a community poll via the WCA website

**Comment by lgarron** [comment]
(Please keep this thread constructive!)

**Comment by pd159** [comment]
To James:
- it would mean anyone could vote, including people who have been cubing for two months, which is bad. This type of decision should be made by more experienced cubers
- not enough people would vote, because it's a simple, optional online poll. It's likely that the people who want it removed won't vote because they aren't extremely passionate about it. On the other hand, pretty much everyone who wants it to stay would vote because they have practiced the event. This will make the results be in favour of the ones who want it to stay and won't reflect community opinion very well

All in all, we should all wait for the higher-ups to make the decision for us, as they're more capable than the average cuber

**Comment by CarterKoala** [comment]
I feel like the community poll could be a good idea if implemented correctly, such as having requirements to vote. That being said, I'm not really sure of a good way to do that (and I've seen the current proposal) but I'd love to find a good way for that to happen.

**Comment by lgarron** [comment]
> * it would mean anyone could vote, including people who have been cubing for two months, which is bad. This type of decision should be made by more experienced cubers
> * not enough people would vote, because it's a simple, optional online poll. It's likely that the people who want it removed won't vote because they aren't extremely passionate about it. On the other hand, pretty much everyone who wants it to stay would vote because they have practiced the event. This creates an obvious bias and won't reflect community opinion very well

These are good points. I think the best way to mitigate this is to:
- have each poll be done by a user with a WCA account (this doesn't require you to have competed, but lets us see how poll responses differ based on who has).
- Include it as a larger poll with more topics and wider circulation.

**Comment by jamesquinn1** [comment]
But should people who don't really care either way be the ones making the decision?
Also, why should someone's whos been cubing for 10 years matter more than a newer cuber?

**Comment by pd159** [comment]
Not saying that you need to be a WCA veteran to vote, but rather that having very new cubers vote is bad. For the most part, they aren't aware of the general state of the community due to a lack of experience with comps, online cubing threads, etc. Consequently, they won't have a certain confidence level needed to accurately form an opinion on what they believe is the better decision.

**Comment by ccbaird** [comment]
3x3 with Feet should remain as an official event not on the basis of its popularity or lack thereof within the community, but on its own merits. 

Most basically, 3x3 with feet is a method of solving a twisty puzzle, specifically a Rubik’s cube. Rules have been standardized, there is no confusion about how to scramble or what constitutes a solved state. It meets the basic criteria for inclusion as a WCA event. Furthermore, because it is already an established event the burden of proof for whether it should remain ought always to be on those advocating change. 

Further criteria for whether an event ought to be an official event, beyond thematic congruity, standardization, and history, include but are not limited to:
- Uniqueness: does it require a different skill set than existing events?
- Competitiveness: are there people who will compete at a high level around the world?
- Crowd appeal: do people like to watch it? Will it bring new competitors in?

Uniqueness: there are two arguments here for maintaining Feet as an official event. First, by limiting TPS Feet rewards move efficiency differently than other WCA events. Obviously the very fastest solves and cubers in all variations employ both dexterity and efficiency, but most cubers rely more on one than the other. Modern hardware and techniques have pushed the balance between the importance of dexterity and the importance of efficiency in One Handed solving very close to the balance for two handed solving. That balance is naturally weighted much more strongly toward efficiency for solving with feet. Whereas at one point cubers like myself who are naturally not as dexterous as others could achieve competitive times by learning to solve efficiently one handed, that is no longer as feasible. Feet offers another way to achieve competitive times with a different skill set, both natural skill and learned skill. This is reflected in another important reason for keeping feet as an official event: there is a different set of people who are good at it. In the WCA database there are only 4 people ranked in the top 100 for average for all 3 of 3x3, OH, and Feet. There are 5 in the top 100 for both OH and Feet, 4 for 3x3 and Feet, and 38 ranked in the top 100 for OH and 3x3. Not only does Feet reward different skills, it has a different set of people who are good at it. The number of different elite individuals is arguably as important as the raw number of people participating, especially in terms of encouraging new talent to strive for excellence. 

Competitiveness: this has clearly increased in the last year. Multiple records broken, 4 people with sub-30 averages and 151 with sub-1:00 averages to 11 sub-30 and 249 sub-1:00.

Crowd Appeal: Feet is different. It’s entertaining. We can’t expect others to take us seriously if we take ourselves too seriously. In my experience not everyone cares to know how to solve a Rubik’s cube, and not everyone who cares to solve a Rubik’s cube cares to do it with their feet, but most people would love to watch somebody solve a Rubik’s cube with their feet. And if they can watch that then taking the time to learn to solve normally is downright normal.

Are there people with hangups about hygiene? Sure, but objectively speaking those fears are groundless. There are people with weird sexual fetishes about feet, which is a drawback, but there are people with sick sexual fetishes about children too, and we quite rightly allow children at competitions. The primary arguments that I have heard against having 3x3 with feet as an official event boil down to “it doesn’t add anything,” “ew, gross,”  and “it doesn’t look professional” To the first, I believe I have adequately explained above why and how that is not true. To the second: no, it’s not gross, grow up. And to the third, quit taking yourself so seriously. You are spending hours playing with what is basically a child’s toy. If you think that’s professional you’re wrong in the head. Fun? Heck yeah, but it’s not supposed to be professional.

**Comment by GenTheThief** [comment]
I would just like to put my voice out there as another Feet supporter. The reasons presented here have been developed well and, I think, outweigh and outnumber the reasons for removing the event.
On the topic of public perception, as I haven't seen many examples yet, I have had several people who found out that I could solve a Rubik's Cube with my feet ask me to show them, and have had one boy ask me to solve his own cube with my feet. They have all been very impressed and thought that it was amazing that I could do that. They haven't seemed particularly grossed out that I'm using my feet.

Also, I want to thank Lucas for giving all of us the updates and clarifications.

**Comment by TheMagicCuber** [comment]
Here is a copy of an email conversation between Lucas Garron and myself about 3x3 WF. Hopefully this can be of help.
Dear WRC,

I’m understand that there is a number of people who are calling for the removal of the event 3x3 With Feet. However, there are many reasons why it should stay as an event

To start, it is more popular than it may seem. There are many individuals in the WCA that may not have competed in feet, but would like to. There are also people who don’t particular enjoy the event, but are not in favor of its removal, as they see the appeal, and see how many people actually enjoy it.

Next, I want to talk about many of the popular complains: one common complain is that it is unsanitary. However, many competitions keep the sanitation of competitions and cubes in mind and take precautions to keep everything clean. In the case of the two competitions I co-organized, when the event came up in the schedule, we used different mats and timers, gave scramblers gloves and wipes, and also gave competitors wipes to wipe their feet.
Another complain is that there is a low number of people who actually compete. This is true, but when compared to an event such as 5x5 blindfolded, the number of competitors for feet is much greater. Although it isn’t the most popular, it still has some very determined individuals in the community (myself included) who love to compete in it. 

One thing on Facebook I saw was comparing the whole event to the removal of Magic and Master Magic. However, unlike those events where a majority of the community was for the removal, many cubes are indifferent or want to keep feet as an event.

I and many others understand that in the end it is your decision as to the outcome of the event, but I know that your responsibility of deciding regulations also relies on the community’s voice and reasons. So with that, I urge you to reconsider removing feet, and hope that you will keep it as an official event. 

Thank you

-Braden

Hi Braden,

Thank you for contacting us! We're listening to all feedback, and will take your points into account as we consider next steps for 3x3x3 With Feet. 😊

»Lucas

My understanding is that the removal of Magics was also fairly controversial. However, I haven't looked back at this in a while. Do you have links that give a good overview of the community reaction to this?

Hello Lucas,
The Cubing Historian has a very good overview on the events of Magic and Master Magic as a whole. 
https://youtu.be/U6n0q-43Vdk
Here is a link to his video “The Death of Magic”.

The reasons for removal are discussed around 1:57, and the community’s reaction was summarized quite shortly at 2:47.

While I wouldn’t compare the community’s reaction for magic to feet, there was definitely a bigger call for the removal of both Magic’s. 
https://www.speedsolving.com/forum/threads/official-confirmation-of-the-removal-of-magic-and-master-magic-events.37815/

This is a link to a speedsolving forum about the removal of Magic. Like with feet, the reaction is mixed, but did show a strikingly positive reaction, as people saw the reasoning to be sound and justified. 
While the reaction for feet is definitely more split, it is definitely comparable.

Hope this helps :) and thank you for taking the time to respond and take interest in my email.

Cheers! 

-Braden

**Comment by TheMagicCuber** [comment]
Also, I want to address the claim that 3x3 WF makes non-competitors take the WCA less seriously, or are appalled by it. While it is a reaction to hearing about it, most people I have spoken to about the event are actually very intrigued and think it is cool. When I tell people I have a sub-45 Average in 3x3WF, the reaction is usually that people are impressed. It is mainly people within the community who do not like the event who say it degrades the WCA as a whole. Some also say that the WCA should be mature and have a good/formal image, and having 3x3 WF as an event ruins this. While this is a good thing to have, the fact that many members of the WCA are children, combined with the fact that speedcubers are, when boiled down to the bare facts, playing with an toy. Now I know that this is controversial, and I don’t think that is just “playing with a toy.” There is a lot of skill that goes into algorithms, memorization, bld solving, etc, that elevates it to way above toy status. 3x3 WF shouldn’t take away from that, as it takes a lot of hard work and skill to solve with your feet fast. With Daniel Rose-Levine and other fast feet-solvers, for example, using optimized algorithms and a specific turning style adds a new element to just normal 3x3. I cannot say with honesty that there are no flaws to WF, or that there are no reasons to remove it. However, I do not believe that the reasons to remove the event outweigh the reasons to keep it. The image of the WCA is already very positive, and I believe that having WF keeps things interesting :) Thank you

**Comment by Tehzeebkohli** [comment]
WF is an ‘alive and kicking’event. Look at the records and crazy numbers. It’s amazing how many are competing so seriously.  Sub 30 averages are quite common now. It clearly shows the amount of passion and practise that has gone into this event. 
It would be wonderful to have the support of the WRC for WF to see just how far this event goes and how low the numbers drop. 
It’s so fascinating to watch (as is blindfolded) and brings a unique perspective to solving the cube. 
People are actually intrigued with WF solving and I’ve yet to meet someone who finds it yucky or gross or ‘not serious’. 
The others reasons have been presented beautifully by everyone on this thread. 
My main reason for keeping WF is- let’s see where this event goes given the proper support. Look at the numbers in this year itself... this event is getting more popular and it’s just so much fun to watch.

**Comment by TheMagicCuber** [comment]
Here is a quote from Daniel Goodman (DGCubes) in a Facebook group:
“Sorry, but the "makes cubing look unprofessional" point is overused and (imo) invalid. Feet gets some of the best reactions from non-cubers; just because you don't like practicing it doesn't mean spectators don't enjoy watching it.

The unfortunate thing is that there's no concrete way to prove either side of this point since it'll always boil down to an anecdotal argument. I do wonder why proponents of removing feet are so fixated on professionalism though. Cubing is inherently a weird/nontraditional hobby; if it's fun to do, why do we care how it looks to the public? (Although I do think that if cubing has a wide enough outreach the public will either disregard it entirely or accept it for what it is, and I don't believe that that would come down to whether or not we solve cubes with our feet.)”

**Comment by SeanMoran27** [comment]
I know people may see me as biased for my views on the event, however I feel my opinion matters regardless.
We already have a restricted 3x3 event in the WCA (One Handed) so it seems unnecessary to have another one such as feet. The only advantage I see with feet over one handed is that non-cubers are fascinated  by it. Apart from that, there isn’t much going for the event IMO. Furthermore, having feet at a competition requires many logistical differences to the rest of the WCA events, for example, you have to change the display setup if you are using one, and place stackmats/timers on the floor, and change the arrangement of chairs. From an organiser’s point of view, this would be too much effort for one event. Another event I feel people complain about holding is clock, with how easily the pins can fall/move, however this can be alleviated if judges/runners are careful while holding the clock, or if there were some new innovation in clock covers. This isn’t the case with feet. I have also heard of many cases of people intentionally +2ing feet solves to avoid doing a double turn, this just seems like a loophole in the regs, and I would be welcome to a way of preventing this is feet is kept as an event. Also I think we shouldn’t completely dismiss the argument that ‘feet is gross’ - yes some people think that, but feet does have a sanitary issue compared to other events, as it brings the risk of verrucas/warts being exposed to the stackmat/timers, which I feel could be a health danger, as with the rest of the WCA events where we use our hands (or hand) we wash them usually multiple times a day, so the risk of this occurring is reduced, and this is not the case with feet.

**Comment by Samuel-Baird** [comment]
Plenty of people just don’t like it ¯\_(ツ)_/¯ despite having almost no barrier of entry feet has the third lowest competitor count of all events (only being best out by two of the hardest events). Not only do people just not compete in it but I have seen so many people over the years ask for fret yo be removed, the only other event I have seen that is clock. In addition plenty of the people who compete in it just do it as a joke, for SOR/kinch or just to get bronze+ memberships. 

And while you could argue that if people don’t like it then they don’t have to compete in it, it does affect others.

Competitors can be required to staff feet even if they find it repulsive, it eats up time at competitions, it cost money to buy the extra feet equipment (which can also be annoying to transport) which means competitors are having to pay more for competitions, and it effects their championships experiences.

Large championships are basically required to hold all events and with championships getting larger and longer the more streamlined they can be the better. In addition to that feet requires floor space which in some conditions can not be achieved without getting an extra room ex. US nationals 2018. It also takes extra time to setup and takedown whereas almost all other events need minimal setup. By removing it you can have more time for other events, potentially add more rounds or use it for buffer time which can help staff feel less pressure and as a result not make as many errors in hastiness. 

It’s unsanitary and can leave an unpleasant odor in the room or on objects that the cube comes in contact with (we have people with BO problems but hands won’t leave BO odor on cubes, timers and mats)

Feet adds almost nothing new to the wca other than manipulation of the cube with your feet.

And last but not least, opinion alert! I feel like feet definitely gives cubing a weirdo/freak show vibe. With speed cubing growing and starting to get more attention I feel like professionalism and spectator friendliness is a route worth going. With more media coverage and large events more money is coming into cubing which is resulting in more growth, by presenting feet it may deter people from covering it in a professional way and turn it into a “look at this weird thing”. Talking with non-cubers I can confirm speedsolving a 3x3, doing it one handed or solving it behind your back is pretty much always seen as amazing or cool (unless they think you know some “trick”) but feet I have always gotten a mix of “cool!”, “gross” and “that’s weird”, people are pretty mixed on it.

**Comment by jamesquinn1** [comment]
One thing that just recently came to mind: The explanation for keeping the outrageous numbers for keeping feet secret was that this would inflate the demand, but while this would be artificial inflation, it would be a growth in popularity nonetheless, and I find that argument pointless.

**Comment by jamesquinn1** [comment]
To respond to your points @UnderwaterCuber 
"Plenty of people .... bronze+ memberships." - The argument about people only do it for unofficial stats is bad, as this is basically saying "people only do it because it's official", which is true for the vast majority of events. Consider the amount of people who do 7x7 to 8x8, or square-1 to square-2. 

About the ways it affects others-
Just wear gloves to staff if you find it repulsive for some reason, which I still don't understand. 
If money is tight in a community you can use the same equipment for hand solving and feet solving. If not, buying 8 timers at $30 a piece is $240, or about $0.25 per registration fee if you spread the cost over 10 comps. This cost is rounding error to nothing.
I've never understood the argument of it takes too long to set up, as I've always found it easier to do as we have done it without displays. Moving timers from the ground to the table takes a maximum of 15 seconds per timer.
On the topic of scheduling time, I have always found feet to be by far the easiest event to schedule, as it is quite easy to run in the back of the room or a side room on other timers while another event is going on. 
I would rather touch a cube that has touched a kids foot 100x more than a hand that is used for picking boogers, and other teenage boy tasks.

Opinion Alert: I have never once had a noncuber describe feet as disgusting. That freak show comment that people (thankfully you didn't) reference so much was made by a cuber. Every noncuber is been in awe that I could solve a Rubik's cube with my feet in just a couple minutes. I've been asked to do feet solves in public, and gathered far larger of a crowd than when I did blind solves, one handed solves, or normal solves.

**Comment by Tehzeebkohli** [comment]
I’ve NEVER in all the competitions I’ve been to have seen anyone with warts/ veruccas on their feet attempt feet solving. Nor have I heard of anyone having witnessed this. 
Perhaps people are unaware that hands could be filthier than feet but that doesn’t stop us from shaking hands (or fist bumping as the kids do) 
People have their personal likes and dislikes. If you don’t like it just don’t do it. 
If kids who don’t like it do it just for bronze/ silver membership.... more power to them. They have put their love for cubing over their mindset and are opening their minds to new avenues. That’s superb in my opinion. 
And most importantly let’s not forget the popularity With feet  has gained and the records that are being broken. 
An event like this needs in my humble opinion needs to be supported to see where it can go. 
Have a great day everyone!!!

**Comment by TheMagicCuber** [comment]
I was told to post this here, because it’s important but also has a lot to offer for not just feet, but how Cubing is viewed in the public. Also I think I’m posting here too much so I’ll limit my posts after this

Also Sorry for weird formatting:

This is in reply to someone’s comments addressing their concerns for feet being an event, which can be viewed on Katie Hull’s post in “The Best Event Group” on Facebook

Here's just a reply to each of your points: -yes, I do agree that some people just dont like it. I can honestly understand why, and I don't think someone's opinion is wrong just because I don't agree with it. It's just an opinion, so if you don't like it, thats ok. However, I do like it, so if you are going to say I can't compete in it just because you don't like it, well, no ¯\_(ツ)_/¯      -I could argue that you should just not compete, but I don't agree with that stance          -Yes, I do agree that feet can be time consuming and expensive, with extra mats being purchased, or timers, or whatever. When Steven and I organize Flag City, and it has feet, Zach Garber uses the same mats, but different timers, and we wash each with clorox wipes that I provide, because I think paying $6 for thing of clorox is worth it to hold an event I like. Sure, some people may not like to scramble, and thats ok. But, there are people competing in the event who, when not solving, will. Thats why at Flag City we also provided gloves for people to use to scramble. So, if you didnt want to solve but were ok with scrambling, then you just wore gloves.              -I do agree that Large championships should be required to hold it, and I do think that, if it puts a burden on the schedule/staff, things should be thought through. However, If only 1 round is needed to be held at a championship, then so be it. At nationals, they just used the same rooms from BigBLD but with a different setup. It did take time, yes, but it was worth it to allow the people who did want to compete, to compete. It may be time consuming, but some people hate having clock because custom covers, scrambling, whatever; that event is time consuming too, and even though it is faster, most championships still have at least 2 rounds of it :-/           - It may be unsanitary, yes, but clorox wipes are a thing that are inexpensive and can be used. Also, as for a smell from the feet, honestly I got nothing. Feet can stink and thats a bad thing. I wouldn't go as far to say it can stink up a room, but a cube maybe, or a mat or timer. But, again, clorox.           - I completely disagree when saying that feet adds nothing to the WCA other than just manipulating the cube with your feet. The slow tps, different techniques, lower movecount and reliance on efficiency are crucial to feet. ZBLL is a popular thing among top solvers for a good reason. Less moves and easy to do with your feet. With the logic of feet adding nothing to the WCA, then the same could be said about OH- its just 3x3 but with one hand, "nothing special". except, it isnt; OH requires new fingertricks and other algs just like Feet, along with techniques and stuff like that. So, I don't think that Feet can be said to add nothing to the WCA, because if it can be said, then OH should be on the chopping block as well... - and lastly, the freakshow vibe thing. I recognize this is your opinion, but is a widely held opinion so I will address it the same way. In my earlier comment about a personal experience on r/pan, I did get a few comments saying that it was weird, but no one said that I shouldn't do it because it was weird. Many people encouraged me to do more feet solves because it was interesting, intriguing, and downright cool to watch. When my local newspaper did a video about the first flag city competition Steven and I organized, Feet had a major part in the video because, although it was seen as different, it was seen as cool, unique, and fun to watch. Even freakshows can be fun to watch. On this same topic, people say that It diminishes the credibility that the Cubing community has. However, I personally don't think it does. If you, as a cuber, had to imagine what a "cuber" would be like, you'd probably imaging someone like Feliks, Mats, Max, or other solvers. However, someone who isn't as invested in the community would imagine a more stereotypical cuber: a kid (probably asian, given stereotypes) who can solve it in like 30 seconds with both hands. The fact that competitions exist can impress many people who aren't in the community, let alone fast solvers. And, when someone sees that people can compete in competitions by doing it with their feet, that is just icing on top of the impressed cake. The cubing community isnt necessarily seen by the general public as a professional thing. It is, there is no doubt, but if literal bronies at Nats 2018 can say that WE'RE the nerds, then there is some hesitation in public opinion. Yes, I do think that cubing should be taken seriously, but that is hard to do for a lot of reasons; one of the main reasons being that a lot of the competitors are kids. It's hard to be a professional organization when most of your competitors are below the age of 18. Removing feet won't solve that issue. What would help with public image more would be more recognition by more "general media", such as maybe BBC or CNN or Fox doing an actual tv segment about the Cubing Community, with it being factual, having interviews with questions other than "Whats your fastest time", and stuff like that. I don't think cubing can ever be mainstream (I'd love to be proven wrong, however), because of the stereotype around solving a rubik's cube: Its impossible unless you're a genius, but I'd love a wider reception of it. Feet can't hurt that, in fact i think it helps; If someone who didn't know cubing was a big thing, saw two articles: one was "Person solves rubik's cube in 4.22 seconds", and the other was "Person solves rubik's cube with feet in 16.96 seconds", I am positive most people would click on the second article.           Wow I typed wayyyy too much. Hope this was helpful, and Katie Hull, (and Lucas Garron) Hope this info helps the discussion

**Comment by TheMagicCuber** [comment]
Also @Tehzeebkohli, to be fair to whomever said the comment about competing with warts or whatever, I competed in feet with one of my toes being in recovery after smashing it with a drum-set. It didn’t look as bad as it had been, since it had been a few months, but definitely not a normal looking toenail

**Comment by Tehzeebkohli** [comment]
@TheMagicCuber  I’ve seen kids eating greasy pizza and just wiping their hands on their T-shirt’s and running for their solve... while their moms scream at them to clean up with a wet wipe. 
It’s all good in my opinion. 
This is something they love and look forward to. Cleaning their hands would probably be last thing on their minds when they are all geared to smash their PB’s 
We live in a polluted world full of germs and bacteria. It’s everywhere. People don’t stop riding the metro coz it’s full of harmful bacteria. Or swimming in a pool in which very possibly some kid may have peed in. 
Why should an event be removed coz a minority find it gross or disgusting?? 
 It’s a personal choice. Don’t like it don’t do it. But don’t stop others who adore this event from doing it or hosting it. 
Just my two cents. 
Also I totally support your statement of solving with feet requires a low tps and knowledge of zbll. You can’t spam tps with your feet and that in itself is remarkable. I am so impressed how kids can get these times looking down at a cube as opposed to in front of them and solve so beautifully using as less moves as they possibly can.

**Comment by Samuel-Baird** [comment]
> @TheMagicCuber I’ve seen kids eating greasy pizza and just wiping their hands on their T-shirt’s and running for their solve... while their moms scream at them to clean up with a wet wipe.
> It’s all good in my opinion.
> This is something they love and look forward to. Cleaning their hands would probably be last thing on their minds when they are all geared to smash their PB’s
> We live in a polluted world full of germs and bacteria. It’s everywhere. People don’t stop riding the metro coz it’s full of harmful bacteria. Or swimming in a pool in which very possibly some kid may have peed in.
> Why should an event be removed coz a minority find it gross or disgusting??
> It’s a personal choice. Don’t like it don’t do it. But don’t stop others who adore this event from doing it or hosting it.
> Just my two cents.
> Also I totally support your statement of solving with feet requires a low tps and knowledge of zbll. You can’t spam tps with your feet and that in itself is remarkable. I am so impressed how kids can get these times looking down at a cube as opposed to in front of them and solve so beautifully using as less moves as they possibly can.

People can wash their hands pretty easily at comps, it’s a bit harder to do that with feet :P I have also had a few comps where competitors had athletes foot and it’s smelled pretty bad.

As for the reason why it should be removed because a “minority” find it gross and you don’t have to be involved with it, you do realize whether or not you want to if you are asked to you can be required to scramble and judge the event and that if you don’t you get DQd from the comp. The only way to truly not be involved is to not go to any competitions with feet lol

**Comment by ccbaird** [comment]
@UnderwaterCuber it is entirely possible to wash your feet. The last time I competed in Feet I judged that my feet were clean enough and I was in a hurry, but the first time I washed my feet before I left home and again in the bathroom sink right before competing. It’s not difficult. 

As far as being required to scramble or judge, has that ever happened? I understand that some people are squeamish about feet (it’s silly, but whatever), and I understand the need for a rule allowing organizers or delegates to compel people to help. I find it very difficult to believe that anyone would ever actually be forced to choose between DNFing their results and judging or scrambling Feet. Surely they could trade places with someone scheduled to work a different event.

**Comment by Tehzeebkohli** [comment]
@UnderwaterCuber let me tell you about an incident which happened very recently. I accompanied my son to a competition where there was clock. The scrambling was getting a bit difficult and the competition was getting slightly delayed. But everybody pitched in to scramble and run for clock being extra extra careful about the pins for the few (minority) serious clock competitors. It totally warmed my heart seeing young children and teenagers giving it their all to see that their clock friends could have a positive environment to compete ❤️
In this intolerant and judgemental world I saw a sliver of hope. Kids supporting each other over a plastic toy, being there for their friends and not judging them for this event that they take seriously. 
It’s the same experience I’ve had with feet. I’ve never ever seen/ heard of someone who is forced to scramble or someone that refuses to scramble or judge, or someone who is disqualified if they prefer not to scramble or judge. Non feeters cheer their with feet friends and stand like videographers all around during solves. The hype has definitely increased multiple gold in this past year. 
The cubing community is so supportive as a whole. From these cubing competitions, slowly slowly the children will enter into the real world. Let’s teach them tolerance, empathy, kindness and love. Let’s teach them to support one another and not tear others down. Let’s teach them that we are a global family even though we may be different. 
Hope you all have a fantastic day/ night ❤️

**Comment by DailyCuber** [comment]
So I have competed in feet once, earlier this year after it was announced to get removed, and that was only because it was an event of the wca. You could say I only did it to get a ranking in that event. (And friends and family definitely looked at it extremely weirdly to the point where I didn't really say I did 3x3 with feet anymore or was kinda embarrassed to say that I did it, but that's completely besides the point.) So first of all, I am noticing everyone talking about the "yuck" factor, though the original post by the wca didn't ever specify anything about that. They said that feet had been requested to be removed multiple times, and that the popularity wasn't so high as to justify keeping it. Thus the reason they are planning on removing it in 2020, also giving enough warning for everyone who does and doesn't do feet. They were also lenient enough to even wait a couple years and to test and see if more people would get involved with it over the long amount of time that they kept it in for. But all in all, I still think the main reason to remove it is as I listed above: Lack of competitors, and multiple requests to have this event removed. (Note that this is coming from an obviously biased opinion on this matter, though I still tried to specify the facts as much as possible.)

**Comment by ccbaird** [comment]
“Friends and family definitely looked at it extremely weirdly to the point where I didn’t really say I did 3x3 with feet anymore or was kinda embarrassed to say that I did it.”

My experience, and that of everyone I’ve talked to about it, has been the exact polar opposite of that. Solving a Rubik’s cube for fun? Weird. Doing it with your feet? Cool!

At the last comp I organized there was a guy who had never done anything with cubing, but heard that something was going on at church so he stopped by to watch. He was having a fine time just hanging out and talking to people, but when we started doing feet he said “I’ve got to see this,” and went to get the best spot to watch. The runner handed him my cube thinking he was the judge, and he would have been happy to judge if he had known the rules. Instead he stepped back to let somebody else judge, and thought the whole thing was crazy and awesome.

**Comment by alexmaasswca** [comment]
My opinion on the removal of feet: honestly compared to magics being removed in 2012, this seems more out of hatred of the event rather than more actual reasons of logistics, magic was removed simply because it was too hard to properly judge it (with timer starts and stops and such)

**Comment by ccbaird** [comment]
@alexmaaswca Is that an argument for or against removing Feet? Some people like it, some people don’t, but it doesn’t have the issues with judging that Magic did.

**Comment by DailyCuber** [comment]
> “Friends and family definitely looked at it extremely weirdly to the point where I didn’t really say I did 3x3 with feet anymore or was kinda embarrassed to say that I did it.”
> 
> My experience, and that of everyone I’ve talked to about it, has been the exact polar opposite of that. Solving a Rubik’s cube for fun? Weird. Doing it with your feet? Cool!
> 
> At the last comp I organized there was a guy who had never done anything with cubing, but heard that something was going on at church so he stopped by to watch. He was having a fine time just hanging out and talking to people, but when we started doing feet he said “I’ve got to see this,” and went to get the best spot to watch. The runner handed him my cube thinking he was the judge, and he would have been happy to judge if he had known the rules. Instead he stepped back to let somebody else judge, and thought the whole thing was crazy and awesome.

Again, my original purpose for posting what I did was that many people are missing the point and noting the hygiene about it rather than the reasons that were listed in the original post. I was just posting my own experience, though I understand that it is very impressive to non-cubers. 

> My opinion on the removal of feet: honestly compared to magics being removed in 2012, this seems more out of hatred of the event rather than more actual reasons of logistics, magic was removed simply because it was too hard to properly judge it (with timer starts and stops and such)

I agree, though a lack of competitors also has a bit of a part in it

**Comment by alexmaasswca** [comment]
> 
> > My opinion on the removal of feet: honestly compared to magics being removed in 2012, this seems more out of hatred of the event rather than more actual reasons of logistics, magic was removed simply because it was too hard to properly judge it (with timer starts and stops and such)
> 
> I agree, though a lack of competitors also has a bit of a part in it
It is improving a lot though, and lack of competitors is also partly because of lack of holding it. 




> @alexmaaswca Is that an argument for or against removing Feet? Some people like it, some people don’t, but it doesn’t have the issues with judging that Magic did.

This is an argument for keeping feet. Its removal just seems based on almost pure dislike rather than actual non-opinion based reasons

**Comment by lgarron** [comment]
This post provides more details to supplement [this announcement](https://www.worldcubeassociation.org/posts/removing-official-status-for-3x3x3-with-feet):

> Dear WCA Community,
> 
> After a full year of feedback and review, the WRC and the WCA Board have come to the conclusion that 3x3x3 with Feet should be removed from the list of official WCA events as [previously planned](https://github.com/thewca/wca-regulations/issues/652), on January 1, 2020. Although it is clear that some people will be significantly impacted, there has not been a significant change in participation in the event throughout 2019. 3x3x3 with Feet remains the least popular speedsolving event by a significant margin, and concerns from organizers about hygiene and logistics remain. 
> 
> We know that this outcome is disappointing for many community members, and understand the frustration that removing an event from the official list will have a significant impact on its popularity. However, any competition is still welcome to hold it as an unofficial event, and organizers may allow competitors with medical considerations to compete using feet during 3x3x3 Speed Solving. (See [Guideline 5b+](https://www.worldcubeassociation.org/regulations/guidelines.html#A5b+) and [Regulation 2s](https://www.worldcubeassociation.org/regulations/#2s).)
> 
> We also welcome suggestions to bridge the gap between unofficial and official events. For example:
> 
> - Making it easy to sign up for unofficial events during official registration.
> - Having a standard place for unofficial results from WCA competitions.
> 
> We would also like to take this time to apologize for the flaws in the process used for this decision. We have learned greatly from the community's feedback, and will prioritize a clear and transparent process for event changes in the future.
> 
> Lucas Garron
> WRC leader
> On behalf of the Regulations Committee
> 


In September 2017, a Delegate poll with the question "Should 3x3x3 With Feet be an official WCA event?" resulted in the following votes:

- Yes: 56
- No: 58

In November 2017, the Board agreed with a proposed plan:

> The format of Feet should be changed from Mean of 3 to Average of 5. One year after the change has come into effect, we will analyze the popularity again. Feet will only be kept as an event if more than 30% of the competitions in the last 12 months held Feet and more than 4.5% of the competitors competing in the last 12 months competed in Feet.

In November 2018, the WRC decided not to remove 3x3x3 with Feet immediately, but to announce a planned removal in 2020.

There was interest in putting this to a vote of Delegates and/or the community again. However, there are logistical issues with trying to get a vote that accurately represents the desires of the entire community, and the WRC has concluded that it’s not the best way to handle the decision at this time.

As of November 2019, there has been a lot of public discussion, but the original targets remain far. The participation percentages for the last few years are:

  | 2013 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019
-- | -- | -- | -- | -- | -- | -- | --
% of competitions | 32% | 28% | 26% | 25% | 23% | 21% | 23%
% of competitors | 4.3% | 3.4% | 3.1% | 2.7% | 2.8% | 2.9% | 3.1%

(The 2019 stats are from the end of October.)

This remains the lowest participation rate of any speed solving event, by a significant margin. The percentages for all other speed solving events are at least 1.5x as large (percentage of competitions) or 2x as large (percentage of competitors).

**Comment by ccbaird** [comment]
So having a community-wide survey, or at least a new survey of all delegates and organizers, was discussed, but as soon as it became clear that only a vocal minority wanted Feet removed that idea was canned? Way to listen to community input.

This will go down as one of the worst decisions that the WCA has ever made, both in terms of the process and in terms of the content. I am very disappointed that the WCA has chosen to eliminate one of my favorite events and even more disappointed that an organization that has historically done a lot of good popularizing and standardizing a really fun activity has allowed itself to be highjacked by killjoys.

**Comment by CarterKoala** [comment]
Also, what's up with the short notice? Yes we knew that it was a possibility, but at least give people an actual time frame before it's gone for good, especially considering the many competitions that are already announced to have feet in 2020.

**Comment by Samuel-Baird** [comment]
> Also, what's up with the short notice? Yes we knew that it was a possibility, but at least give people an actual time frame before it's gone for good, especially considering the many competitions that are already announced to have feet in 2020.

The original notice was a year ago and I feel like they have made it clear that there wouldn’t be any guarantees that feet was going to stay

**Comment by CarterKoala** [comment]
> > Also, what's up with the short notice? Yes we knew that it was a possibility, but at least give people an actual time frame before it's gone for good, especially considering the many competitions that are already announced to have feet in 2020.
> 
> The original notice was a year ago and I feel like they have made it clear that there wouldn’t be any guarantees that feet was going to stay

Saying it “might” be removed is not a decision.

**Comment by alexmaasswca** [comment]
![image](https://user-images.githubusercontent.com/48773372/70103664-bd059d80-1609-11ea-9a68-2ac479485597.png)
lmao

**Comment by alexmaasswca** [comment]
lol
![78147483_106687140816325_6629426839083286528_n](https://user-images.githubusercontent.com/48773372/70103704-dd355c80-1609-11ea-8ec3-c01b4e9ae6ee.jpg)

**Comment by Tehzeebkohli** [comment]
This is probably the worst and most bias decision ever. What happened to the community poll? Isn’t WCA a community to promote cubing or am I mistaken?? 
Seriously I’m shocked and highly disappointed by how this has been handled.

**Comment by abunickabhi** [comment]
Hello everyone,

My association with Feet has been quite less, with me competing in it in 2015 to complete all 18 events, and then participating it in 2019 since it was going to be last year of this event. To be honest, I found feet a good experience and it was the only cubing event in much I enjoyed speedcubing in general (Like experiencing the solving process, finding the joy in completing the cube). Events like pyraminx, 2x2 and Skewb finish in a few seconds not giving the speedcubing joy, and all other events were where I was constantly trying to improve. Blind events are my main events so I have tremendous respect for these events especially 5BLD and MBLD. The event that I enjoyed the least is the Rubik's clock to be honest. I competed in it only once, and I have never bought a clock puzzle in my 11 years of association with speedcubing.

It feels a bit awkward to have only 17 events in the WCA now, and I hope the next event with least amount of competitor participation does not get removed. The next twisty puzzle to get introduced should be chosen in a more democratic and transparent way since our community is very listening and selfless (eg SS Forums, WCA Forums). Board members please do take a note of that!

Happy Cubing!
Cheers,
Abhijeet Gokar
WAC Community Representative

**Comment by DhruvA219** [comment]
Cubing as a sport was meant to be enjoyable but removing an event like feet just because of some people complaining about illegitimate issues with it really shows how hate can really ruin the enjoyment for people who actually invested their time in this event. 

The reason that 3x3 Feet was different and some organisers wanted it out was enough to convince the WRC and Board that this event needs to be removed. I would like to know any reason apart from the popularity issue or about how ‘some people’ (unrelated with the event) feel  - which led to this big decision.

The event doesn’t have any regulatory issues, it has no genuine problems in organising and above all these, this unique event is enjoyable to those who practice it. And that’s what cubing is all about. We don’t solve for the public, or for the arbitrary numbers that is the number of competitions or competitors. We solve different events in different ways because its fun, and removing an event that was reaching new heights in terms of competing and records, seeing it being removed for unjust and false reasons is sad. However, I’m just a mere speckle in terms of the community and to the WCA and I understand that even though this may not change what the WRC thinks, I still think it is important to point these things out.

I did get my one and only national record in this and now realising that all my practice in the last 3 years have all gone to waste saddens me to the point where I don’t even feel like practising any event. 
In a community with no voice and a governing body which has become closed and centred, I believe the golden years of speedcubing have come to an end and the days when cubing was more about the experience are over. The ever-growing hate and toxicity for certain events really shows what it has all come down to. The cubing community with its selfless love for what started as a mere ‘toy of the year’  to a passion sport has now reached a point where people are becoming self-centred and toxic and are ruining the experience for the rest of us.

**Comment by Tehzeebkohli** [comment]
I wonder what happened to the “community wide survey” seeing as this is supposed to be a “community” 
Can anyone throw some light on this please.

**Comment by Tehzeebkohli** [comment]
> So having a community-wide survey, or at least a new survey of all delegates and organizers, was discussed, but as soon as it became clear that only a vocal minority wanted Feet removed that idea was canned? Way to listen to community input.
> 
> This will go down as one of the worst decisions that the WCA has ever made, both in terms of the process and in terms of the content. I am very disappointed that the WCA has chosen to eliminate one of my favorite events and even more disappointed that an organization that has historically done a lot of good popularizing and standardizing a really fun activity has allowed itself to be highjacked by killjoys.

I really wonder what happened to this community wide survey seeing that it is supposed to be a cubing “community” 
With all due respect this has been handled extremely badly.

**Comment by Swapnil5384** [comment]
3x3x3 is the most common and most practised cube worldwide. It would not be fair to remove [3x3x3 cube. ](https://www.cubelelo.com/) # https://www.cubelelo.com/
